src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/arm/conv_fq.cpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_test_utils/node_builders/convolution.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "utils/cpu_test_utils.hpp"
+#include "utils/fusing_test_utils.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/fake_quantize.hpp"
+#include "openvino/op/matmul.hpp"
+
+using namespace CPUTestUtils;
+
+namespace ov {
+namespace test {
+class ConvAndFQ : virtual public SubgraphBaseTest, public CPUTestsBase {
+public:
+    void SetUp() override {
+        abs_threshold = 4e-3f;
+        targetDevice = ov::test::utils::DEVICE_CPU;
+
+        std::tie(inFmts, outFmts, priority, selectedType) = CPUSpecificParams{{}, {}, {}, CPUTestsBase::any_type};
+        const auto precision = element::f32;
+        const auto input_static_shape = Shape{4, 3, 2, 2};
```

Comment:
Can we make `shape` a test parameter? We should at least cover one more case: dynamic shapes